### PR TITLE
Fix message for box command on default help

### PR DESCRIPTION
**Pull request by brgnepal**
Changing the help message for box command to make it scalable.

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ en:
 
           Commands:
                env          displays connection information for services in the box
-               box          displays version and release information for the box
+               box          displays box related information like release version, ip etc
```
Comment (navidshaikh) [inline locales/en.yml]: Nice, thanks!

Just add a comma like and rephrase like
displays box related information like version, release, IP etc

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+- Fix message for box command on default help @budhrg
```
Comment (navidshaikh) [inline CHANGELOG.md]: We are pushing the new changelogs at the bottom of Unreleased section to avoid the conflicts, no?

Comment (brgnepal) [inline CHANGELOG.md]: Ya I forgot :)

**Comment by brgnepal** [comment]
@navidshaikh Updated PR as per feedback

**Comment by navidshaikh** [comment]
LGTM!

ping @bexelbie

**Comment by bexelbie** [comment]
LGTM

**Pull request merged**